tests/create-job.executor.test.ts

```
@@ -0,0 +1,1134 @@
+
+    const job = await getJobStatus(jobId);
+    expect(job!.status).toBe("failed");
+    expect(job!.error).toContain("OpenRouter API error");
+    expect(job!.error).toContain("429");
+  });
+
+  test("template ownerAccountId matches job.ownerAccountId, not hardcoded ADMIN_ACCOUNT_ID", async () => {
+    installHappyPathMocks();
+    const { executeCreateJob } = await import(
+      "../src/api/v2/agent-templates/services/job-executor"
+    );
```

ESLint

[error] Replace `·await·import(⏎······"../src/api/v2/agent-templates/services/job-executor"⏎····` with `⏎······await·import("../src/api/v2/agent-templates/services/job-executor"`

(prettier/prettier)